modules/ROOT/pages/gov-cloud-features.adoc

```diff
@@ -5,7 +5,17 @@ endif::[]
 
 MuleSoft Government Cloud includes the full functionality of the Anypoint Platform runtime plane (services and Mule runtime engine) and a subset of the Anypoint Platform control plane (browser-based management of APIs and Mule runtime engine instances) functionality. Government Cloud supports the full API-led connectivity methodology and end-to-end API lifecycle management.
 
-MuleSoft Government Cloud supports Mule runtime engine v3.9 and later.
+MuleSoft Government Cloud supports Mule runtime engine v3.9 and later, with some restrictions:
+
+* Because MuleSoft Government Cloud does not support Mule agents with self-signed certificates, specific versions of Mule agents are required:
+
+** Mule 3: v1.12.0 and later (Supported only for cloud deployments)
+** Mule 4: v2.2.0 and later
+
+* For FIPS compliance, Mule 4 agent version 2.4.9 or later is required.
```

Comment:
this is only for standalone mules. can we remove this line and include it in the subpage?

Reply:
Moved this.